src/components/LogUpdate/CriticalCarePreview.tsx

```diff
@@ -49,14 +52,27 @@ export default function CriticalCarePreview(props: Props) {
 
   return (
     <div className="w-full transition-all duration-200 ease-in-out md:mx-auto md:max-w-5xl md:pt-8">
-      <div className="py-4">
+      <div className="py-4 flex items-center justify-between max-[368px]:flex-col max-[368px]:items-start">
         <ButtonV2
           id="back-to-consultation"
           variant="secondary"
           href={`/facility/${props.facilityId}/patient/${props.patientId}/consultation/${props.consultationId}`}
         >
           {t("back_to_consultation")}
         </ButtonV2>
+        <Link
+          className="mt-2"
+          href={`/facility/${props.facilityId}/patient/${props.patientId}/consultation/${props.consultationId}/log_updates/${props.id}/update`}
+        >
+          <Button
+            id="update-log"
+            className="bg-primary text-white"
```

Comment:
set the variant prop to primary instead

Reply:
![image](https://github.com/user-attachments/assets/28486627-8881-4823-96d3-1e255ee7ff58)

Outline_Primary?

Reply:
You can either use that, or copy the `primary` implementation from: https://github.com/ohcnetwork/care_fe/pull/9076/files#diff-c29ec4c06502c45a9ee8740156618a5823f7cadbcd01a339304a66537c0dc30dR14-R15

Reply:
> set the variant prop to primary instead

@nihal467  rithvik asked me to do so